openpype/hosts/maya/api/plugin.py

```diff
@@ -226,7 +226,17 @@ def update(self, container, representation):
             if alembic_nodes:
                 for attr in alembic_attrs:
                     node_attr = "{}.{}".format(alembic_nodes[0], attr)
-                    alembic_data[attr] = cmds.getAttr(node_attr)
+                    connections = cmds.listConnections(node_attr, plugs=True)
+                    data = {
+                        "connected": False,
+                        "attribute": None,
+                        "value": cmds.getAttr(node_attr)
+                    }
+                    if connections:
+                        data["connected"] = True
+                        data["attribute"] = connections[0]
```

Comment:
I feel like this could be rewritten to:

```data = {
    "input": next(iter((cmds.listConnections(node_attr, plugs=True, destinations=False) or []), None))
    "value": cmds.getAttr(node_attr)
}
```

So basically no need to track `connected` + `attribute` separately since if input is None you know it wasn't connected. You can just store `input` (== `attribute`) as single value.